module/VuFind/src/VuFind/Navigation/AccountMenu.php

```diff
@@ -172,6 +172,13 @@ public static function getDefaultMenuConfig(): array
                         'icon' => 'search',
                         'checkMethod' => 'checkHistory',
                     ],
+                    [
+                        'name' => 'usercomments',
```

Comment:
Equivalent configuration should also be added to [AccountMenu.yaml](https://github.com/vufind-org/vufind/blob/dev/config/vufind/AccountMenu.yaml).